test.rb

```diff
@@ -0,0 +1,16 @@
+class AAA
+  def initialize
+    @a
+  end
+
+  def a
```

Comment:
__[development][rubocop]__
* Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

<!-- Comment by [SideCI](https://sideci.com) -->